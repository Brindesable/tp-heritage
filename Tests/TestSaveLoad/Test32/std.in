C circle1 0 0 10
SAVE a.txt