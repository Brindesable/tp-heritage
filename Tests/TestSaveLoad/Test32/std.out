OK
#1 object(s) created
OK
